stylesheet.css

```diff
@@ -0,0 +1,257 @@
+body {
+  height: 100vh;
+  width: 100vw;
+  margin: 0;
+  background-image: url("images/galaxy.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center center;
+  font-family: "Space Mono", monospace;
+}
+.header {
+  color: white;
+  text-align: center;
+  font-family: "Space Mono", monospace;
+  font-size: 5rem;
+}
+
+.selectPlayer {
+  color: white;
+  text-align: center;
+  font-family: "Space Mono", monospace;
+  font-size: 2em;
+}
+
+#startButton {
+  /* background-color:white; */
+  background: transparent;
+  color: white;
+  font-size: 1.5rem;
+  padding: 0.6rem 1rem;
+  margin-bottom: 1.5rem;
+  cursor: pointer;
+}
+
+#startButton:hover {
+  color: black;
+  background: white;
+}
+
+.main-container {
+  height: 100%;
+  width: 100%;
+  /* background-image: url("images/galaxy.jpg"); */
+  background-position: center center;
+  background-size: cover;
```

Comment:
always toggle your styles in the inspector and remove any styling that is inconsequential, such as here.